From: Roy Buitenhuis <roy.buitenhuis94@gmail.com>
Subject: [PATCH] package-config generation bug, wrong format used in meson.build

https://bugzilla.gnome.org/show_bug.cgi?id=795730

diff --git a/libgit2-glib/meson.build b/libgit2-glib/meson.build
index bab4f61..62a7eba 100644
--- a/libgit2-glib/meson.build
+++ b/libgit2-glib/meson.build
@@ -220,7 +220,7 @@ pkg.generate(
   description: 'libgit2-glib, a a glib wrapper library around the libgit2 git access library.',
   filebase: libgit2_glib_api_name,
   subdirs: libgit2_glib_api_name,
-  requires: 'libgit2 >= @0@, glib-2.0 >= @1@, gobject-2.0 >= @1@, gio-2.0 >= @1@'.format(git2_req, glib_req),
+  requires: ['libgit2 >= ' + git2_req, 'glib-2.0 >= ' + glib_req, 'gobject-2.0 >= ' + glib_req, 'gio-2.0 >= ' + glib_req],
   variables: 'exec_prefix=${prefix}',
   extra_cflags: extra_args
 )
