From 14f31ee20b231ee59732eae15c8cc241ededc4d5 Mon Sep 17 00:00:00 2001
From: Christian Hergert <chergert@redhat.com>
Date: Sat, 4 Jan 2020 11:03:20 -0800
Subject: [PATCH] gvls: fix vapi install dir

---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 2888d6c..e47bfb8 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -1,4 +1,4 @@
-vapidir = join_paths (vala_dep.get_pkgconfig_variable ('datadir'),'vala','vapi')
+vapidir = join_paths (get_option('datadir'),'vala','vapi')
 GIR_NAME= VERSIONED_CAMEL_CASE_NAME+'.gir'
 TYPELIB_NAME= VERSIONED_CAMEL_CASE_NAME+'.typelib'
 VAPI_NAME = VERSIONED_PROJECT_NAME+'.vapi'
-- 
2.23.0

